rasa_nlu_examples/featurizers/dense/bpemb_featurizer.py

```diff
@@ -355,19 +356,18 @@ def train(
             for attribute in DENSE_FEATURIZABLE_ATTRIBUTES:
                 self.set_bpemb_features(example, attribute)
 
-    def create_word_vector(self, document):
+    def create_word_vector(self, document: Text) -> np.ndarray:
         encoded_ids = self.model.encode_ids(document)
         if encoded_ids:
             return self.model.vectors[encoded_ids[0]]
 
         return np.zeros((self.component_config["dim"],), dtype=np.float32)
 
-    def set_bpemb_features(self, message: Message, attribute: Text = TEXT):
+    def set_bpemb_features(self, message: Message, attribute: Text = TEXT) -> None:
         text_vector = self.create_word_vector(document=message.text)
         word_vectors = [
             self.create_word_vector(document=t.text)
-            for t in message.data["tokens"]
-            if t.text != "__CLS__"
```

Comment:
This was actually incorrect before. In case of `responses` we would still take the `text` tokens.